packages/git/git.1.4.0/opam

```diff
@@ -28,6 +28,7 @@ depends: [
   "hex"
   "conduit" {>= "0.6.0"}
   "ocamlbuild" {build}
+  "camlp4"
```

Comment:
Aha, this is the old `sexplib.syntax` use, so needs `sexplib {<="113.00.00"}` and `type_conv`, which will pull in the  unified extension.